lib/code-caller/index.js

```diff
@@ -28,27 +28,31 @@ const { PythonCaller } = require('./code-caller-python');
 let pool = null;
 
 module.exports = {
-  init() {
+  async init() {
     debug('init()');
 
     const { workersExecutionMode } = config;
     if (workersExecutionMode !== 'container' && workersExecutionMode !== 'native') {
       throw new Error(`unknown config.workersExecutionMode: ${workersExecutionMode}`);
     }
 
+    if (config.workersExecutionMode === 'container') {
+      await initCodeCallerDocker();
+    }
```

Comment:
I moved this from `server.js` to here so that the server startup routine doesn't need to be directly aware of execution modes, etc.

Reply:
Much nicer!